server/controllers/oidc.js

```diff
@@ -0,0 +1,102 @@
+const axios = require("axios");
+const { v4 } = require('uuid');
+const { getService } = require("@strapi/admin/server/utils");
+
+const configValidation = () => {
+  const config = strapi.config.get('plugin.strapi-plugin-sso')
+  if (config['OIDC_CLIENT_ID'] && config['OIDC_CLIENT_SECRET'] && config['OIDC_ISSUER'] 
```

Comment:
`OIDC_ISSUER` does not seem to be used, so I would like to remove it if it is not a problem

Reply:
Sure go ahead.